cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -145,6 +147,48 @@ func packedPolicy(req *pluginapi.ContainerPreferredAllocationRequest) []string {
 	return deviceIds
 }
 
+// Returns a slice of by-path Mounts for a cardPath&Name.
+// by-path files are searched from the given bypathDir.
+// In the by-path dir, any files that start with "pci-<pci addr>" will be added to mounts.
+func bypathMountsForPci(cardPath, cardName, bypathDir string) []pluginapi.Mount {
+	linkPath, err := os.Readlink(cardPath)
+	if err != nil {
+		return nil
+	}
+
+	// Fetches the pci address for a drm card by reading the
+	// symbolic link that the /sys/class/drm/cardX points to.
+	// ../../devices/pci0000:00/0000:00:02.0/drm/card
+	// -------------------------^^^^^^^^^^^^---------.
+	pciAddress := filepath.Base(strings.TrimSuffix(linkPath, filepath.Join("drm", cardName)))
+
+	if pciAddress == "" {
+		return nil
+	}
```

Comment:
Aren't all GPUs supposed to have PCI address?  If yes, I think this should log something, maybe return error to caller too (but I guess that would require updating more tests).

Reply:
> Aren't all GPUs supposed to have PCI address?

I would assume so, but I haven't checked/verified.

As for logging, since the `Scan()` function is ran repeatedly, I didn't want to spam here. I'll think about logging it once.